src/hotspot/share/oops/constantPool.cpp

```diff
@@ -1612,27 +1612,27 @@ void ConstantPool::copy_operands(const constantPoolHandle& from_cp,
 // Copy this constant pool's entries at start_i to end_i (inclusive)
 // to the constant pool to_cp's entries starting at to_i. A total of
 // (end_i - start_i) + 1 entries are copied.
-void ConstantPool::copy_cp_to_impl(const constantPoolHandle& from_cp, int start_i, int end_i,
-       const constantPoolHandle& to_cp, int to_i, TRAPS) {
+void ConstantPool::copy_cp_to_impl(const constantPoolHandle& from_cp, int start_cpi, int end_cpi,
```

Comment:
The comment needs updating now the parameter names have changed.

Again I don't think there was any need to change the names in these cases.